src/Azure.Deployments.Extensibility.Providers.Kubernetes/V2/Data/K8sApiCatalog.csv

```diff
@@ -0,0 +1,94 @@
+Group,Version,Kind,Plural,Namespaced,MajorMinorServerVersions

```

Comment:
Ideally, we should automate the generation of this file as part of the k8s provider type package generation process in CI, which is something to investigate in the future.